### DISCUSSION TITLE
LAW36 - Definition without direct specification of 𝐸 (Young's Modulus)

### DISCUSSION THREAD

**Discussion by Jakub-Motl** [OpenRadioss/OpenRadioss#3218] [Q&A]
Hello everyone!

I’m having an issue with a simulation using the LAW36 material. If I don’t specify E directly and only provide stress-strain curves, the simulation doesn’t run and there are no errors or warnings in the output file. I’m using Open Radioss build 20240315.

Is it even possible to define LAW36 this way?

Thank you in advance for any response.
Jakub Motl

![image](https://github.com/user-attachments/assets/19adb1a6-8eed-4b62-9bdb-b68531489a9b)

**Comment by MarianBulla** [comment]
Hello @Jakub-Motl,
and thanks for you interest in OpenRadioss.

Thank you also very much for this catch. 
I will adress it, so there will be an error on that, if the user will not provide it.
In LAW1, there is alread an error, if the Youngs' modulus is not provided.

LAW36 (as most of the elasto-plastic material laws) start in the elastic domain, where the material behaves pure elastic.
For this reason, one must define the Youngs' modulus (or modulus of elasticity): E-Modulus.

This is a mandatory input value as the Poisson's ratio and density.

For LAW36, you also need to define the hardening curve as a table:

--> True-Plastic-Strain vs. True-Stress.

Please have a look into the examples in the manual page, which provide some example cuves.

There is also some training material available on how to compute these curves (which can also be strain-rate dependant, as you did it).
For this, you also need one E-Modulus.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Schwoertzig** [comment]
Hi @Jakub-Motl ,
Young modulus E is mandatory to describe behavior in elastic domain.
Provided functions are for plastic behavior. You should have fct(0) > 0 (it is yield stress)
Regards

**Reply by Jakub-Motl** [reply]
Hi @MarianBulla @Schwoertzig ,

 Thank you both for your answers! I would also like to ask if it is possible to specify the E modulus as a function of the strain rate using LAW36 or another material model?

Thank you in advance!
Jakub

**Reply by MarianBulla** [reply]
Hi @Jakub-Motl,
yes, this is a very good point, especially for polymeres. 
For metals (where this material was initally developed for), this effect is not observed.
Anyway, we are working on such material law, which will become a new material law.

For now, I would suggest you to switch to a HyperElastic material law, using the Prony-Series, to model the viscosity effect on the elastic behavior. Here, LAW42 using only one parameter pair represents the Neo-Hooken material law, two pairs model the Mooney-Rivlin material and more do the Ogden law. Here you can add the viscosity in the elastic domain, by using the Prony values.
There is a lot of literature for how to do the parameter identification. Or you use directly your curves with our material LAW69 (have a closer look into the manual for it, please).

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Schwoertzig** [comment]
Hi @Jakub-Motl ,
no law 36 is not a **visco**-elasto-plastic model. Young modulus is not depending on strain rate. Maybe law65 should help to do what you want.
For your information I just introduced a Starter check about Young modulus in law36 : https://github.com/OpenRadioss/OpenRadioss/pull/3219
thank you for your feedback !
Regards

**Comment by Jakub-Motl** [comment]
Thank you @Schwoertzig and @MarianBulla 

Yes, polymers, that's exactly where I'm aiming! A new material law is great news.

Thank you again and best regards.
Jakub

**Reply by MarianBulla** [reply]
So, maybe you can already have a look in our LAW69. 
Maybe, it will already help you.
Thanks and best regards,
Marian
